packages/flutter/test/material/switch_list_tile_test.dart

```diff
@@ -100,4 +100,47 @@ void main() {
     await tester.tap(find.byType(SwitchListTile));
     expect(value, isFalse);
   });
+
+  testWidgets('SwitchListTile contentPadding', (WidgetTester tester) async {
+    Widget buildFrame(TextDirection textDirection) {
+      return MediaQuery(
+        data: const MediaQueryData(
+          padding: EdgeInsets.zero,
+          textScaleFactor: 1.0,
+        ),
+        child: Directionality(
+          textDirection: textDirection,
+          child: Material(
+            child: Container(
+              alignment: Alignment.topLeft,
+              child: SwitchListTile(
+                contentPadding: const EdgeInsetsDirectional.only(
+                  start: 10.0,
+                  end: 20.0,
+                  top: 30.0,
+                  bottom: 40.0,
+                ),
+                secondary: const Text('L'),
+                title: const Text('title'),
+                value: true,
+                onChanged: (bool selected) {},
+              ),
+            ),
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildFrame(TextDirection.ltr));
+
+    expect(tester.getSize(find.byType(SwitchListTile)), const Size(800.0, 126.0)); // 126 = 56 + 30 + 40
```

Comment:
// 126 = 56 + 30 + 40

Maybe say where these 3 numbers come from? One looks like app bar height?

Reply:
These were left from another test I used to make this one, but they aren't actually important to the test since the test is just checking that the padding does the right thing. I removed them